lib/job.jsonnet

```diff
@@ -44,6 +45,7 @@ function(jobName, agentEnv={}, stepEnvFile='', patchFunc=identity) patchFunc({
     BUILDKITE_PLUGIN_K8S_WORKDIR: std.join('/', [env.BUILDKITE_BUILD_PATH, buildSubPath]),
     BUILDKITE_PLUGIN_K8S_JOB_TTL_SECONDS_AFTER_FINISHED: '86400',
     BUILDKITE_PLUGIN_K8S_JOB_BACKOFF_LIMIT: '0',
+    BUILDKITE_BUILD_CREATOR_TEAMS: '',
```

Comment:
This seems unrelated! I guess this is related to #59?

Reply:
Yes. If you’re not part of a team, the plugins crashes. If I remember correctly teams are a paid feature, so this fixes the plug-in for anyone on the free tier.

Reply:
Done